mediathread/projects/views.py

```diff
@@ -390,6 +390,36 @@ class SequenceAssignmentEditView(AssignmentEditView):
     template_name = 'projects/sequence_assignment_edit.html'
 
 
+class SequenceEditView(LoggedInCourseMixin, ProjectReadableMixin,
+                       TemplateView):
+
+    template_name = 'projects/sequence.html'
+
+    def get(self, request, *args, **kwargs):
+        """
+        A multi-panel editable view for the specified project
+        Legacy note: Ideally, this function would be named project_view but
+        StructuredCollaboration requires the view name
+        to be  <class>-view to do a reverse lookup
+
+        Panel 1: Parent Assignment (if applicable)
+        Panel 2: Project
+        Panel 3: Instructor Feedback (if applicable & exists)
+
+        Keyword arguments:
+        project_id -- the model id
+        """
+        project = get_object_or_404(Project, pk=kwargs.get('project_id', None))
+        show_feedback = kwargs.get('feedback', None) == "feedback"
```

Comment:
Feedback and the notion of a "response" are applicable only to the assignment structure. These can be removed here.